tools/hypestv/src/windows/hyperv.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! Functions for interacting with Hyper-V VMs.
+
+use anyhow::Context as _;
+
+/// Runs hvc with the given arguments.
+pub fn run_hvc(
+    f: impl FnOnce(&mut std::process::Command) -> &mut std::process::Command,
```

Comment:
Why did you factor this so that the caller has to know the argument syntax for hvc? It seems like it would be preferrable to move any details about hvc into this file.

Reply:
I didn't feel that an extra layer of abstraction was valuable here, given the limited scope of this project. If the scope grows significantly or if the details get more complicated (e.g., if we switch to using WMI directly instead of hvc), then I can imagine adding more methods here.